pkg/controller/manage/kmesh_manage.go

```diff
@@ -210,9 +210,6 @@ func shouldEnroll(client kubernetes.Interface, pod *corev1.Pod) bool {
 	if strings.EqualFold(pod.Labels[constants.DataPlaneModeLabel], constants.DataPlaneModeKmesh) {
 		return true
 	}
-	if pod.Annotations[constants.KmeshRedirectionAnnotation] == "enabled" {
```

Comment:
@lec-bit This fix would break the secret manager assumption, which heavily rely on this to count numbers

Reply:
ok，i will test and Modify